lib/school.rb

```diff
@@ -0,0 +1,49 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names,
+              :time_array,
+              :eod
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    student_names << name
+  end
+
+  def end_time
+    @time_array = start_time.split("")
+    @eod = time_array[0].to_i + hours_in_school_day
```

Comment:
we shouldnt store this as an instance variable, just return what you need to return